ios/ci_scripts/ci_post_clone.sh

```
@@ -18,14 +18,23 @@ fi
+    echo "No .nvmrc file found, using default Node.js version: $NODE_VERSION"
+fi
+
+nvm install $NODE_VERSION
+nvm use $NODE_VERSION
+nvm alias default $NODE_VERSION
 
 # Verify Node.js version
 node -v
 
-sudo ln -sf $(which node) /Users/local/.nvm/versions/node/v18.17.0/bin/node
+sudo ln -sf $(which node) /Users/local/.nvm/versions/node/$NODE_VERSION/bin/node
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)